docs/airgap.md

```diff
@@ -41,3 +41,13 @@ i.e., `localhost,127.0.0.1,0.0.0.0,10.0.0.0/8,longhorn-system,cattle-system,catt
 When the nodes in the cluster do not use a proxy to communicate with each other, the CIDR needs to be added to `http-proxy.noProxy` after the first node is installed successfully. Please refer to [fail to deploy a multi-node cluster](./troubleshooting/harvester.md#fail-to-deploy-a-multi-node-cluster-due-to-incorrect-http-proxy-setting).
 
 :::
+
+## Image Packaging
```

Comment:
Since the section most addresses guest cluster images, how about naming the section "Guest Cluster Images"?